generator/src/html/script/Main.hx

```diff
@@ -18,9 +19,34 @@ class Main {
 		// parse and locate myselft
 		assert(tocData != null, "toc data is missing");
 		var toc = J(JQuery.parseHTML(tocData));
-		assert(document.URL.startsWith(document.baseURI));
-		var myUrl = document.URL.replace(document.baseURI, "").replace(window.location.hash, "");
-		if (myUrl == "")
+        
+        var baseURI = null;// document.baseURI ;
+        //FIXME IE workaround for the lack of document.baseURI 
+        if (baseURI == null) {
```

Comment:
@aszasz the point was to _never_ break, since we couldn't possibly commit  (or, at least, merge) something that didn't even work on a modern browser :wink: